packages/@coorpacademy-components/src/molecule/learning-profile-radar-chart/index.js

```diff
@@ -0,0 +1,395 @@
+import React, {Fragment, useCallback, useEffect, useMemo, useState} from 'react';
+import PropTypes from 'prop-types';
+import {
+  Radar,
+  RadarChart,
+  PolarGrid,
+  PolarAngleAxis,
+  ResponsiveContainer,
+  Tooltip,
+  PolarRadiusAxis
+} from 'recharts';
+import {
+  find,
+  pipe,
+  keyBy,
+  mapValues,
+  size,
+  get,
+  getOr,
+  map,
+  toPairs,
+  values,
+  isEmpty,
+  omit,
+  fromPairs,
+  times,
+  flatten
+} from 'lodash/fp';
+import classnames from 'classnames';
+import {isMobile as getIsMobile} from '../../util/check-is-mobile';
+import style from './style.css';
+
+const uncappedMap = map.convert({cap: false});
+
+// TICK_POSITIONS
+const top = {offset: {x: -100, y: -65}, alignment: 'center', margin: 'auto'};
+const bottom = {offset: {x: -100, y: 10}, alignment: 'center', margin: 'auto'};
+const right = {offset: {x: 30, y: -10}, alignment: 'start', marginRigth: 'auto'};
+const left = {offset: {x: -230, y: -10}, alignment: 'end', marginLeft: 'auto'};
+
+// CONSTANTS
+const DEFAULT_COLORS = {
+  gradient: {
+    fill: ['#0062ffff', '#0062ffff'],
+    stroke: ['#0062ffff', '#0062ffff ']
+  },
+  percentage: {
+    color: '#000000ff',
+    background: '#ffffff'
+  },
+  label: {
+    color: '#000000ff'
+  }
+};
+
+const CHART_CONFIGS = {
+  triangle: {
+    name: 'triangle',
+    ticks: [top, right, left],
+    sideCount: 3
+  },
+  quadrilateral: {
+    name: 'quadrilateral',
+    ticks: [top, right, bottom, left],
+    sideCount: 4
+  },
+  pentagon: {
+    name: 'pentagon',
+    ticks: [top, right, right, left, left],
+    sideCount: 5
+  },
+  hexagon: {
+    name: 'hexagon',
+    ticks: [top, right, right, bottom, left, left],
+    sideCount: 6
+  }
+};
+
+const CHART_CONFIGS_BY_SIDE_COUNT = pipe(keyBy('sideCount'), mapValues('name'))(CHART_CONFIGS);
+
+const CUSTOM_DOT_DEFAULT_PROPS = {
+  strokeWidth: 4,
+  strokeOpacity: 0.2,
+  fill: '#fff',
+  r: 8,
+  pointerEvents: 'all',
+  style: {cursor: 'pointer'}
+};
+
+const CUSTOM_DOT_ACTIVE_PROPS = {
+  fill: '#fff',
+  r: 8,
+  strokeWidth: 6,
+  strokeOpacity: 0.5
+};
+
+const RADAR_DEFAULT_PROPS = {
+  strokeWidth: 6,
+  strokeOpacity: 0.2,
+  fillOpacity: 0.2
+};
+
+// COMPONENTS
+const Gradient = ({type, colors: [firstColor, secondColor]}) => (
+  <defs>
+    <linearGradient id={`gradient-${type}`} x1="0%" y1="0%" x2="0%" y2="100%">
+      <stop offset="0%" stopColor={firstColor} />
+      <stop offset="100%" stopColor={secondColor} />
+    </linearGradient>
+  </defs>
+);
+
+const CustomDot = ({cx, cy, payload, onDotClick, activeDot, dataKey, stroke}) => {
+  const {value: activeDotValue, label: activeDotLabel} = activeDot;
+  const label = get('payload.subject', payload);
+  const value = get(`payload[${dataKey}]`, payload);
+  const isCurrentDotActive = value === activeDotValue && label === activeDotLabel;
```

Comment:
maybe inline (au formatage pret)

```suggestion
  const isCurrentDotActive = get(`payload[${dataKey}]`, payload) === activeDot.value && get('payload.subject', payload) === activeDot.label;
```